### ISSUE TITLE
Nvidia Shield TV Pro Kodi Nexus 20 Final cant Install ZIP Repo

### ISSUE THREAD

**Issue by Starfoxfs** [xbmc/xbmc#22483]
### Describe the bug
Here is a clear and concise description of what the problem is:

If i install a fresh version of Kodi Nexus 20 Final on Nvidia Shield TV Pro 2019 (Software Version 9.1.1)
i get at first Kodi 20 Final startup 3 Options for access management
"Allowed only on use the App"
"Allowed for 1 Time"
"Disallowed"

If i click "Only allowed when using the app" i can´t install 3rd Party Repositories like Kodinerds Repo because Kodi can´t find the .ZIP File 

The only way to install this is go to Nvidia Shield Settings -> App settings -> Kodi -> RightManagement and set it to "Allow every Time"
The Nvidia Shield WARNS in this case, because the access to Media is allowed every time even if Kodi is closed.

The Problem here is for the normal User you can´t Install a 3rd Party Repo ZIP with Standard Installation.

## Expected Behavior
Here is a clear and concise description of what was expected to happen:

If i allow 3rd Party Sources i can find the ZIP Files in "install from ZIP"

## Actual Behavior

At the moment i can allow 3rd Party Sources but i can´t find the ZIP Files in "Install from ZIP" if i search thru the Directorys

## Possible Fix

Fix the Access Management on Nvidia Shield TV Pro Update 9.1.1

### To Reproduce

1.  Use a new Nvidia Shield TV Pro
2. Clean Install Kodi 20 Nexus Final
3. At first startup u can only use for Access Managment "Only allowed when using the app"
4. Go to System Settings allow 3rd Party Addons Sources
5. Try to install Kodinerds Repository ZIP thru Install from ZIP, you can´t find this ZIP File.

### Debuglog

The debuglog can be found here:

### Screenshots 
Here are some links or screenshots to help explain the problem:

## Additional context or screenshots (if appropriate)
Here is some additional context or explanation that might help:

### Your Environment
Used Operating system:

 - [ 9.1.1] Android Nvidia Shield TV Pro
 - [ ] iOS
 - [ ] tvOS
 - [ ] Linux
 - [ ] macOS
 - [ ] Windows
 - [ ] Windows UWP

 - Operating system version/name:
 - Kodi version: 20 Final (armv8a)

*note: Once the issue is made we require you to update it with new information or Kodi versions should that be required.
Team Kodi will consider your problem report however, we will not make any promises the problem will be solved.*

**Comment by matke-84** [comment]
Exactly the same problem. Xiaomi Stick 4K, Android 11.

**Comment by CastagnaIT** [comment]
At least on Shield to temporary fix the problem you have to
open app info panel of kodi app, go to kodi app authorizations,
and on "file and multimedia content access" you change it to "allow always"
and at next kodi startup you can see all files

For example VLC app can allow set to "allow always" at first app startup,
but i have no idea if that android dialog can be implemented also on Kodi

**Comment by Starfoxfs** [comment]
@CastagnaIT  yes same here, i have opened "file and multimedia access" and changed it manually to "allow always" 
But this is tricky for a "normal" User who don´t know this and use only the Standard Installation of Kodi

**Comment by joseluismarti** [comment]
VLC app on Android TV also does not allow to select the "Allow all the time" option when requesting permission at startup, this is due to change in permission management introduced by Google.

On stock Android, both apps request the new "Access to all files" permission and we don't need to adjust the permission later.
